shell/run-all-b2b-performance-tests.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+source shell/functions.sh
+
+# Generate the output file
+reportFile=$(create_report_file)
+outputFolder=$(create_report_folder)
+reportFile="$outputFolder/$reportFile"
+
+$(create_folder_if_not_existant "$outputFolder")
+
+# Create arrays for file lists
+filesDirectory1=($(find "tests/b2b/$app/tests" -name '*-test.js' -type f))
+
+# Merge the arrays
+files=("${filesDirectory1[@]}" "${filesDirectory2[@]}")
+
+# Iterate over the merged array
+for file in "${files[@]}"
+do
+  # Get the path of the current file relative to the original directory
+  testFile=${file#$(pwd)/}
+
+  # Construct the docker command
+  command=$(build_k6_docker_command "$testFile" "$reportFile")
+
+  echo "Running command: '$command'"
+
+  # Run k6 on the current file using its original path
+  eval "$command"
+done
```

Comment:
a lot of code is repeated, we will need to adjust the scripts in future

Reply:
I'm aware of that, but I wanted to get it to work for now. The only tricky part is always that those commands are executed from where you currently are in the file system. My understanding is, that when you switch the folder SOMEWHERE in the scripts, it will affect the follow-up called methods. Before I start refactoring this I want to have a stable base. But I'm already planning to do so. I even considered using php or another script language, because I think at some point bash might hit a limit. But then I would have to install php in the docker container or add a php container etc... :)